shell/platform/darwin/graphics/FlutterDarwinContextMetal.mm

```diff
@@ -33,32 +33,39 @@ - (instancetype)initWithMTLDevice:(id<MTLDevice>)device
 
     if (!_device) {
       FML_DLOG(ERROR) << "Could not acquire Metal device.";
-      [self release];
       return nil;
```

Comment:
Can we decorate translation units with `FLUTTER_ASSERT_ARC` and `FLUTTER_ASSERT_NON_ARC` now that we are mixing the two in the engine?

Reply:
Added it to the ARC targets that i've touched.